logs_intake/socket/writeToTLSSocket.js

```diff
@@ -0,0 +1,34 @@
+tls = require('tls');
+var host = "intake.logs.datadoghq.com"
+var ssl_port = 10516
+
+// callback for when secure connection established
+function connected(stream) {
+    if (stream) {
+      // socket connected
+      // REPLACE THE API KEY with your own:
+      api_key= "<your_api_key>"
+      stream.write(api_key + " " + '{"ddsourcecategory": "aws", "appName": "morrisons-xxwmm-storefinder-prod", "clientRequestResource": "/stores/373/specialopeningtimes", "timeBackend": 6, "clientRequestQuerystring": "apikey=kxBdM2chFwZjNvG2PwnSn3sj6C53dLEY", "backendResponseCode": 200, "clientRequestTime": 1524836411874, "clientRequestVerb": "GET", "clientRequestSize": 0, "apiProduct": "location-v2-prod", "ddsource": "apigee", "ddtags": "x.info,level:info,x.api-identity-v1,x.api:identity-v1,x-env-dev,x-env:dev,x.proj-apigee,x.proj:apigee", "clientRequestUri": "/location/v2/stores/373/specialopeningtimes", "environment": "prd", "responseCode": 200, "clientIp": "91.208.239.126", "timeTotal": 9}'+"\n");
```

Comment:
there appears to be some sensitive strings in this, apiKey or whatever. can we scrub that?

Also, this is a very specific log event being sent. for miscelany scripts, we want to make it easy for people to understand what each part of the content is, so we should probably fill it with dummy info. Like "message": "put your log line here", "ddsourcecategory": "optionally add a source category here", etc.

breaking each attribute onto its own line would probably help make it clearer too.